src/components/SequencerSettings.js

```diff
@@ -0,0 +1,51 @@
+import styled from 'styled-components';
+import close from '../images/close.svg';
+
+export default function SequencerSettings({
+  isSettingsVisible,
+  currentBpm,
+  setCurrentBpm,
+  setIsSettingsVisible,
+}) {
+  return (
+    <>
+      {isSettingsVisible ? (
+        <SettingsContainer>
+          <CloseButton aria-label='close' type='button' onClick={() => setIsSettingsVisible(!isSettingsVisible)}>
```

Comment:
Wouldn´t it be better to alwas set isSettingsVisable to false with the close button?

Reply:
i think it doesn't matter, because i can only click the close button if the is state is true...